ci-values.yaml

```diff
@@ -0,0 +1,16 @@
+useInterpodAntiAffinity: false
+
+global:
+  subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
+  tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+  environment: aat
+dummyConfigInCI: true   #Need this for pipleine build only
+definitions:
+ - https://github.com/hmcts/ccd-data-store-api/raw/master/src/aat/resources/CCD_CNP_27_AUTOTEST1.xlsx
+definitionFilename: cmc-ccd.xlsx
```

Comment:
This feature `definitionFilename` only works if the files are baked into the docker image at root level. I don't think latest definition-importer image has these files. May be we should remove this feature completely and offer only path based property "definitions" ?

Reply:
ci-values is used to pass the pipeline, in actual scenario in CMC, they supply file at root level

Reply:
Ok, found the config. It seems clients (CMC) have to build their own wrapper image on top of core definition-importer image and bake their definition files at root level. 
https://github.com/hmcts/cmc-ccd-domain/blob/master/definition/xlsx2json.Dockerfile#L5